src/common/plugins/CircuitAnalysisBases.py

```diff
@@ -82,8 +86,6 @@ def _initialize(self, circuit: dict) -> None:
         self._ground_pins = set()
         self.nodes_count = 0
         self.pin_labels = dict()
-        circuit_name = self.core.get_attribute(circuit, "name")
-        self.pyspice_circuit = Circuit(circuit_name)
         self._log_debug(f"Initialized empty PySpice circuit for {circuit_name}")
```

Comment:
I think this log information is not necessary now.

Reply:
ah, fair :)